rockchip/generic: add missing config symbols

Rondom opened the pull request:
Refresh kernel config to add unanswered config options.

Rondom committed:
rockchip: add missing config symbols

Run make kernel_oldconfig add unanswered config symbols.

Signed-off-by: Andreas Gnau <rondom@rondom.de>
Link: https://github.com/openwrt/openwrt/pull/20121
Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>

hauke commented on target/linux/rockchip/armv8/config-6.12:
```diff
@@ -424,13 +423,15 @@ CONFIG_MTD_CMDLINE_PARTS=y
 CONFIG_NET_DEVLINK=y
 CONFIG_NET_DEVMEM=y
 CONFIG_NET_DSA=y
+# CONFIG_NET_DSA_KS8995 is not set
```
Please move this to `target/linux/generic/config-6.12`. It was added in 73504d0b27847e70f0cdb2acddd0599c7c9c6cc2

Rondom replied:
fixed

Rondom committed:
generic: 6.12: add missing symbol NET_DSA_KS8995

Add missing KConfig symbol CONFIG_NET_DSA_KS8995.

Fixes: 73504d0b2784 ("kernel: kmod-dsa-ks8995: Backport DSA patches")
Signed-off-by: Andreas Gnau <rondom@rondom.de>
Link: https://github.com/openwrt/openwrt/pull/20121
Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>
